### PR TITLE
feat(demos): WasCC demo for AssemblyScript

### LINKED ISSUE #102: Create wascc demo in AssemblyScript
This example should have the code we use to compile the module and a pod spec that loads user specified env vars, and env vars from the downward API/config maps/secrets. These env vars should be printed out via an http request to the pod

### PR DESCRIPTION
**Pull request by bacongobbler**
closes #102 

Signed-off-by: Matthew Fisher <matt.fisher@microsoft.com>

### CODE REVIEW

File: demos/wascc/hello-world-assemblyscript/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "scripts": {
+        "graphql-lint": "graphql-schema-linter",
+        "gql2as": "wapc-gql2as schema.gql > assembly/module.ts && prettier --write assembly/module.ts",
+        "codegen": "npm run graphql-lint && npm run gql2as",
```
Comment (radu-matei) [inline demos/wascc/hello-world-assemblyscript/package.json:5]: Running this fails:

```
npm run codegen                                  

> @ codegen /Users/radu/projects/src/github.com/deislabs/krustlet/demos/wascc/hello-world-assemblyscript
> npm run graphql-lint && npm run gql2as


> @ graphql-lint /Users/radu/projects/src/github.com/deislabs/krustlet/demos/wascc/hello-world-assemblyscript
> graphql-schema-linter



✔ 0 errors detected


> @ gql2as /Users/radu/projects/src/github.com/deislabs/krustlet/demos/wascc/hello-world-assemblyscript
> wapc-gql2as schema.gql > assembly/module.ts && prettier --write assembly/module.ts

sh: wapc-gql2as: command not found
```

Not exactly sure where one can get that utility from.

Comment (bacongobbler) [reply demos/wascc/hello-world-assemblyscript/package.json:5]: So this took me a while to hunt down... it’s hiding under https://github.com/wapc/as-codegen/tree/feature/initial_implementation (note the non-master branch). You have to compile from source.

I’ll add that to the docs

File: demos/wascc/hello-world-assemblyscript/k8s.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hello-world-wascc-assemblyscript
+  annotations:
+    deislabs.io/wascc-action-key: MA6DECXXYRIRPWYJAIKQT7ZBPUCJ7HRBLK3ZSDTGE7WWIJ6VU52PZOCI
```
Comment (thomastaylor312) [inline demos/wascc/hello-world-assemblyscript/k8s.yaml:6]: Just a heads up that we are removing the annotation in @bketelsen's upcoming PR in favor of signed modules

Comment (bketelsen) [reply demos/wascc/hello-world-assemblyscript/k8s.yaml:6]: to be clear, all modules had to be signed already, so we're just removing the extra step and unnecessary annotation

Comment (bacongobbler) [reply demos/wascc/hello-world-assemblyscript/k8s.yaml:6]: okay. Let's cross that bridge when we get there

Comment (bacongobbler) [reply demos/wascc/hello-world-assemblyscript/k8s.yaml:6]: oh wait we merged that PR. So the recommendation is to remove this annotation now?

Comment (thomastaylor312) [reply demos/wascc/hello-world-assemblyscript/k8s.yaml:6]: Yes, the annotation can go 😄

**Pull request merged**